doc/installation.md

```diff
@@ -36,6 +36,10 @@ You can override any configuration values with `-D` option.
 To change the http(default 8080) and grpc(default 8080) ports used by Collector, you can change the settings below.
  - `-Darmeria.port`: http, grpc uses 8080 by default
 
+To change the default value of 10MB for the max message size of gRPC in Collector, change the setting below.
```

Comment:
```suggestion
To change the the max message size of gRPC in Collector, change the setting below. 
```

Reply:
fixed it in 755bb3d91980ac24c9572a060398d1c60a1b5bd7